### ISSUE TITLE
Import Interest date

### ISSUE THREAD

**Issue by rubenverhoef** [firefly-iii/firefly-iii#3129]
I am running Firefly III version 5.0.4

**Description**
I want to import an interest date using CSV. When selecting the row as any of all date entries it will not become availible at the `Interrest date` field of the transaction.
The format of the date is Ymd like this: `20191024`

Any idea how i can import the interrest date, or any other date so it will be visible at the transaciton overview as additional of the normal `date` entry?

**Comment by JC5** [comment]
That should happen just like you describe it. I'll verify.

**Comment by JC5** [comment]
Will be fixed in the next release.

**Comment by rubenverhoef** [comment]
Thanks a lot!
One final question, I can select several date's while importing data from CSV. But when I have an transaction there is only the field interest date (and the normal date). What does the import does when I select an other date, like `Transaction due date`? Does it create an additional field at the transaction overview? Or does it nothing? 
The same question can also asked be for all other fields I can select from CSV import (SEPA Batch ID, and so on).

**Comment by rubenverhoef** [comment]
Sorry my bad, i see the dates were not working and you fixed it with your commit. And the SEPA data etc. is just availible as a field when i select it. 
Only thing i see is that i cannot edit the field after import or add it myself when i create a transaction my own.

**Comment by JC5** [comment]
Make sure you enable field in your preferences.

**Comment by rubenverhoef** [comment]
Great, then i have the date fields which i can fill my own!

But how to add or edit the `SEPA Batch ID` for instance ?

**Comment by JC5** [comment]
You can't, unfortunately. Those are imported but are read-only.

**Comment by rubenverhoef** [comment]
Okay thanks for the explanation. I don't really need it, it was only a question. Maybe something for the future?

**Comment by JC5** [comment]
Who knows, it's on my list :)

**Comment by FelikZ** [comment]
Hello JC5!

I have found that this fix actually not backwards compatible with my (and I guess others) import flow.

Since the release, when I use standard ABN-amro import settings which includes interest-date, it actually started to treat as is it suppose to be but as a result, unique rows not unique anymore.

So when I have imported the same day transactions, I got silently duplicates:
![image](https://user-images.githubusercontent.com/1080269/78125178-eea04100-7410-11ea-80c5-82ea3f54385c.png)
![image](https://user-images.githubusercontent.com/1080269/78125227-05469800-7411-11ea-920d-9dc232c21896.png)

Note the difference: interest date.

Luckily, I have backups, so I can revert/fix/re-apply import. But that could have a big impact for some.

Thanks
